pantab/tests/test_roundtrip.py

```diff
@@ -18,6 +19,14 @@ def assert_roundtrip_equal(result, expected):
         expected["object"] = expected["object"].astype("string")
         expected["non-ascii"] = expected["non-ascii"].astype("string")
 
+    if compat.PANDAS_120:
+        # convert float32, float64, and Float32 to Float64
```

Comment:
Hmm I don't think this is desired behavior. For very large data frames the overhead of Float64 versus float64 can be rather untenable.

I guess I'm not sure about the reading side of things yet. Makes sense on writing to support these but what is your reasoning for preferring Float64 when reading back in versus float?

Reply:
Was mainly based on your comment in the discussion to maintain consistency with the other dtypes: 
https://github.com/innobi/pantab/issues/130#issuecomment-825151612
I have very little need to read from an extract so I don't have a strong opinion here.

Reply:
Hmm yea but only use Float when dealing with Nullable columns. The way I am (perhaps incorrectly) reading this is that we are always using Float even with non-Nullable. Is that true? If not I think I would just want to express this differently to prevent that bug from introducing itself in a future refactor

Reply:
`_types.py` maps non-nullable floats to regular pandas floats. We can't roundtrip a non-nullable float since all pandas floats are nullable, so I added a column for non-nullable floats is in `test_reads_non_writeable`.

Perhaps the comment here could be clearer.